src/core/swap/swap-api.js

```diff
@@ -123,16 +129,30 @@ export async function fetchSwapQuote(
  * Picks the best quote out of the available choices.
  * Exported so we can unit-test it.
  */
+
 export function pickBestQuote(
   quotes: EdgeSwapQuote[],
   preferPluginId: string | void,
-  promoCodes: EdgePluginMap<string>
+  promoCodes: EdgePluginMap<string>,
+  preferType?: EdgeSwapPluginType
 ): EdgeSwapQuote {
   return quotes.reduce((a, b) => {
+    // Prioritize transfer plugin:
+    if (a.swapPluginType === 'transfer') return a
+    if (b.swapPluginType === 'transfer') return b
```

Comment:
The transfer plugin is a hack that never really worked right, and we already plan to replace it with a transfer button on the QR modal.

I would be OK with `if (a.pluginId === 'transfer')`, since that doesn't leak this deprecated / unwanted feature out into the public type definitions.

Reply:
Ok. I'll change to that. Even with the improved transfer GUI, I don't want to remove the transfer plugin as doing so could route same-kind asset transfers through a swap partner and incur unnecessary fees.